pkg/transform/kubernetes/kubernetes.go

```diff
@@ -334,7 +334,7 @@ func populateEnvFrom(app *spec.App) error {
 	return nil
 }
 
-func CreateK8sObjects(app *spec.App) ([]runtime.Object, error) {
+func CreateK8sObjects(app *spec.App) ([]runtime.Object, []string, error) {
```

Comment:
I think that it would be good to add comment explaining what this does and what each return value is (`[]runtime.Object`, `[]string`)